### ISSUE TITLE
Train a model

### ISSUE THREAD

**Issue by David-Estevez** [roboticslab-uc3m/textiles-hanging#6]
Train a model with keras (VGG16)

**Comment by jgvictores** [comment]
Note that we'll be looking at a regression model, so may be a bit more like:
https://github.com/PacktPublishing/Deep-Learning-with-Keras/blob/9279c3e972dd0b5a9cfff46597701db9640e2f2f/Chapter07/air-quality-regression.py
(also note that this example uses the Functional API, but we can use a Sequential model as in https://github.com/PacktPublishing/Deep-Learning-with-Keras/blob/master/Chapter03/keras_VGG16.py)

**Comment by David-Estevez** [comment]
Data Generators implemented via 8820574dba4a4f82318c08b4cdb6cf0d71a459e2

Results from training the same model with generators:
![image](https://user-images.githubusercontent.com/1451819/53950571-b5da4f00-40cc-11e9-8a96-0dc0e9d11435.png)

Loss in the graph is about 1.31 (it can't be easily deduced due to scale, but keras returns that value via command line).

I couldn't get activations feedback as Tensorflow won't compute activation histograms with a validation data generator, but it seems to be happening the same thing as with the "normal" dataset: activations of the first layer are stuck at the same value. Possible causes: Gradient exploding? Bad learning rate? Bad input data?

**Comment by David-Estevez** [comment]
This has been done, but we suspect imbalance is behind the poor classification results: 

              precision    recall  f1-score   support

      Hanged       0.59      0.18      0.28       832
       Floor       0.75      0.95      0.84      2168

So I'm closing this issue and  opening new issues to deal with the imbalance in the dataset.
